pages/learn/more/anti-patterns.md

```diff
@@ -39,7 +39,7 @@ If the device type uses flash memory or an SD card, any write-heavy workload wil
 Any modifications made to the host OS before flashing or at runtime carry undue risk. If the host OS has been modified in some way (other than connection details), any subsequent behavior cannot be guaranteed.
 
 ### Modifying the host’s systemd services via D-Bus
-While {{ $names.os.lower }} provides lots of flexibility in terms of interacting with the host OS via D-Bus, with great power comes great responsibility. Since these interactions can potentially cause {{ $names.os.lower }} devices to disconnect from {{ $names.cloud.lower }} or even fall offline, these API calls should be carefully guarded and considered in the application code.
+While {{ $names.os.lower }} provides lots of flexibility in terms of interacting with the host OS via D-Bus, with great power comes great responsibility. Since these interactions can potentially cause {{ $names.os.lower }} devices to disconnect from {{ $names.cloud.lower }} or even fall offline, these API calls should be carefully guarded and considered in the fleet code.
```

Comment:
```suggestion
While {{ $names.os.lower }} provides lots of flexibility in terms of interacting with the host OS via D-Bus, with great power comes great responsibility. Since these interactions can potentially cause {{ $names.os.lower }} devices to disconnect from {{ $names.cloud.lower }} or even fall offline, these API calls should be carefully guarded and carefully considered.
```